scss/atoms/input_styles/_input_styles.scss

```diff
@@ -0,0 +1,121 @@
+/*
+* ==========================================================================
+* Input field styles
+* ==========================================================================
+*/
+
+@import
+	"dist/chassis",
+	"functions",
+	"mixins";
+
+/*
+* ==========================================================================
+* Normal state styling
+* ==========================================================================
+*/
+
+
+input[type=text],
+input[type=url],
+input[type=tel],
+input[type=number],
+input[type=color],
+input[type=email],
+input[type=search],
+input[type=password] {
+	font-family: "Source Sans Pro";
```

Comment:
This should pull from a variable font, and we currently do not use "Source Sans Pro" anywhere